tpm2tools/pcr_test.go

```diff
@@ -90,3 +90,32 @@ func TestGetPCRCount(t *testing.T) {
 		t.Errorf("tpm simulator has unexpected number of PCRs: %v instead of 24", pcrCount)
 	}
 }
+
+func TestCheckContainedPCRs(t *testing.T) {
+	rwc := internal.GetTPM(t)
+	defer CheckedClose(t, rwc)
+
+	truth, err := ReadPCRs(rwc, FullPcrSel(tpm2.AlgSHA256))
+	if err != nil {
+		t.Fatalf("Failed to Read PCRs: %v", err)
+	}
+
+	toBeCertify, err := ReadPCRs(rwc, tpm2.PCRSelection{Hash: tpm2.AlgSHA256, PCRs: []int{1, 2, 3}})
```

Comment:
nit: Name this toCertify or toBeCertified.

Reply:
done